std_msgs/String[] connection_between
float32 distance